cmd/infracost/cmd_test.go

```diff
@@ -0,0 +1,83 @@
+package main_test
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"github.com/infracost/infracost/cmd/infracost"
+	"github.com/infracost/infracost/internal/config"
+	"github.com/infracost/infracost/internal/testutil"
+	"github.com/stretchr/testify/require"
+	"path/filepath"
+	"regexp"
+	"testing"
+)
+
+var timestampRegex = regexp.MustCompile(`(\d{4})-(\d{2})-(\d{2})(T| )(\d{2}):(\d{2}):(\d{2}(?:\.\d*)?)((-(\d{2}):(\d{2})|Z| [A-Z]{3})?)`)
```

Comment:
```suggestion
var timestampRegex = regexp.MustCompile(`(\d{4})-(\d{2})-(\d{2})(T| )(\d{2}):(\d{2}):(\d{2}(?:\.\d*)?)(([\+-](\d{2}):(\d{2})|Z| [A-Z]{3})?)`)
```

It was working with `2021-08-30T15:04:15-01:00` (with -) but not `2021-08-30T15:04:15+01:00` (with +)